### PR TITLE
Adding Repair_auxdb.rst

### PR DESCRIPTION
**Pull request by VenkyOS**
Signed-off-by: Venkataraman Padmanabhan <venkataraman.padmanabhan@seagate.com>

### CODE REVIEW

File: doc/Repair_auxdb.rst [hunk 1]
```diff
@@ -0,0 +1,118 @@
...
+- Fetch: Loads the next batch of records into the iterator and updates the iterator state to correctly position for the next call.
+
+- Reload: Reload the records from the current position, because the map   may have been altered by an intervening call to add or delete a record.
+
+- Check for end: Determines if the record in the specified position will exhaust the iterator.
+
+Typical Usage
+==============
+
+.. image:: Images/usage.PNG
```
Comment (huanghua78) [inline doc/Repair_auxdb.rst:109]: Don't use IMAGE for code.
Originally, this is a piece of code.

Comment (huanghua78) [reply doc/Repair_auxdb.rst:109]:
```C
struct c2_dbenv mydbenv;
struct c2_addb_ctx myaddb_ctx;
struct c2_cobfid_map mymap;
struct c2_cobfid_map_iter myiter;
uint64_t container_id;
struct c2_fid file_fid;
struct c2_uint128 cob_fid;

/* initialize mydbenv */
/* create or open the map */
rc = c2_cobfid_map_init(&mymap, &mydbenv, &myaddb_ctx, "mycobfidmapname");

/* insert records */
rc = c2_cobfid_map_add(&mymap, container_id, file_fid, cob_fid, &mydbtx);

...

/* enumerate */
rc = c2_cobfid_map_container_enum(&mymap, container_id, &myiter, &mydbtx);
while ((rc = c2_cobfid_map_iter_next(&myiter, &container_id, &file_fid, &cob_fid, &mydbtx)) == 0) {
        /* process record */
}

/* cleanup */
c2_cobfid_map_fini(&mymap);
```

File: doc/Repair_auxdb.rst [hunk 2]
```diff
@@ -0,0 +1,118 @@
...
+
+***************
+Interfaces
+***************
+
+#. Insert record to cobfid_map - executed on every ioservice when a file gets created.
+
+#. Delete record from cobfid_map - executed on every ioservice when a file gets deleted.
+
+#. Enumerate devices/containers (In future, device id will begeneralized to container id) - Used by storage agents (typically storage-in agent to iterate over fid-cob_fid mapping to select nextcob for repair).
```
Comment (huanghua78) [inline doc/Repair_auxdb.rst:66]: s/begeneralized/be generalized/

File: doc/Repair_auxdb.rst (hunk 1 above)
Comment (huanghua78) [reply doc/Repair_auxdb.rst:109]: Change all "c2_" prefix to "m0_" prefix.

File: doc/Repair_auxdb.rst [hunk 3]
```diff
@@ -0,0 +1,118 @@
+============
```
Comment (huanghua78) [inline doc/Repair_auxdb.rst:1]: In general, this document is out-dated.
But we can use this an example for document conversion.

Comment (VenkyOS) [reply doc/Repair_auxdb.rst:1]: Okay

File: doc/Repair_auxdb.rst (hunk 1 above)
Comment (VenkyOS) [reply doc/Repair_auxdb.rst]: Incorporated

File: doc/Repair_auxdb.rst (hunk 2 above)
Comment (VenkyOS) [reply doc/Repair_auxdb.rst]: Incorporated

**Review by mukul-seagate11 — APPROVED** [review]
looks good

**Comment by huanghua78** [comment]
There are two things need to be fixed:
* Remove the image. The image link has been removed, but the image itself is still in the patch.
* To embed a block of code, please use the following format. (I have stated this in this and other reviews):
>```C
> some C code.
>```

This will give us a better format.

**Review by huanghua78 — APPROVED** [review]

**Pull request merged**